gatsby/src/templates/situations/page.tsx

```diff
@@ -37,10 +37,16 @@ const Page: React.FC<IProps> = ({ data, pageContext }) => {
         htmlLanguage={pageContext.langCode}
       />
       <SchemaComp
-        datePublished={data.situation.valid_from}
+        datePublished={
+          data.situation.valid_from
+            ? data.situation.valid_from
```

Comment:
Should we rather preffer `creatted`? 🤔

Reply:
I`m not sure 🤔

Reply:
Reason for this form is that some of the situations have the value of `valid_from` set to a date before it was created. But I don't know if it's somehow relevant that validity for this structured data or there should be the real `created` date.

Reply:
i think we can leave it like this for now and if it will be needed we can edit it later.